spec/features/webauthn/sign_in_spec.rb

```diff
@@ -41,24 +31,47 @@
   it 'does not allow the user to sign in if the hardware button has not been pressed' do
     mock_webauthn_verification_challenge
 
-    sign_in_user(webauthn_configuration.user)
-    click_button t('two_factor_authentication.webauthn_use_key')
+    sign_in_user(user)
+    click_webauthn_authenticate_button_and_cancel
```

Comment:
when skimming this diff I had trouble telling the difference between `_and_cancel` and `_and_verify` because they're the same number of characters, what if we dropped `_and_verify` since that's kind of default behavior and left `_and_cancel` as the differentiated option?

Reply:
Yeah, that seems like a simple way to differentiate. I'll update.

Reply:
I am not super in love with the block approach but can't think of anything better 🤷